pkg/virt-controller/services/template.go

```diff
@@ -151,6 +151,7 @@ type templateService struct {
 	virtClient                 kubecli.KubevirtClient
 	clusterConfig              *virtconfig.ClusterConfig
 	launcherSubGid             int64
+	dockerSELinuxMCSWorkaround bool
```

Comment:
Seems redundant.

Reply:
You mean SELinux + MCS? This is also a bit long to type (since it's also the feature gate), so I'm open to alternative name suggestions!

Reply:
I think you are not using this field.